notebook/static/notebook/js/outputarea.js

```diff
@@ -336,7 +336,7 @@ define([
             if (needs_height_reset) {
                 that.element.height('');
             }
-            that.element.trigger('resize');
+            that.element.trigger('resize.Output');
```

Comment:
What I was after was fixing the bug where 'resize' events trigger up, then are thrown on the window, and are considered *window* resize events.

We could just as easily change the name from 'resize' to 'resizeOutput'. Thoughts?